CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.3.0 (2020-12-02)
+- Added modern theme styling for ag-Grid. [#83](https://github.com/blackbaud/skyux-ag-grid/pull/83)
```

Comment:
Should `ag-Grid` be `the data entry grid component` since that's what we call it on the docs site? Same question about the next entry. We have multiple instances of `ag-Grid` in the changelog, but we also have instances of `data entry grid`. I'm not when one is appropriate and when the other is.

Reply:
I'll use data entry grid for this one since its talking about the component, but the next entry is talking about the ag-grid libraries themselves so I'll leave those